### DISCUSSION TITLE
Contest name support (added)

### DISCUSSION THREAD

**Discussion by ofadam** [magicbug/Cloudlog#756] [General]
It would be helpful to have the CONTEST_ID field in ADIF imports appear when looking at QSO details.

Additionally, could there be a field to set that parameter in the new contest logging module in Cloudlog?

**Comment by magicbug** [comment]
Yep needs adding..

Anyone bored enough to change https://adif.org/311/ADIF_311.htm#Contest_ID into a html form selection list

**Comment by dg9vh** [comment]
Do you mean somethink like that (see attached).

[optionslist.txt](https://github.com/magicbug/Cloudlog/files/5710707/optionslist.txt)


73 de Kim
DG9VH

**Reply by magicbug** [reply]
Yes thanks 👍🏻

**Comment by AndreasK79** [comment]
Something like this?
![image](https://user-images.githubusercontent.com/6977712/102666910-5ad1f900-4188-11eb-90e3-8d562c641518.png)

**Reply by ofadam** [reply]
This looks fantastic!

**Reply by magicbug** [reply]
Now been merged, just need to add it to the QSO popup and edit boxes

**Comment by magicbug** [comment]
Contest ID now shows up in the QSO popup as well
